src/components/calcite-rating/calcite-rating-resources.ts

```diff
@@ -1,4 +1,4 @@
 export const TEXT = {
   rating: "Rating",
-  stars: "stars: ${num}"
+  stars: "Stars: ${num}"
```

Comment:
Can you update the `@default` JSDoc string in rating.tsx too?
https://github.com/Esri/calcite-components/blob/master/src/components/calcite-rating/calcite-rating.tsx#L67